evmcore/tx_validation_test.go

```diff
@@ -86,20 +89,23 @@ func TestValidateTx_RejectsTxBasedOnTypeAndActiveRevision(t *testing.T) {
 	}
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			err := validateTx(test.tx, test.configure(getTestTransactionsOption()))
-			require.Equal(t, ErrTxTypeNotSupported, err)
+			validateOpts, netOpts := getTestTransactionsOption()
+			err := validateTx(test.tx, validateOpts, test.configure(netOpts))
+			require.ErrorContains(t, err, "network validation failed")
```

Comment:
Are you sure the test condition for what has caused the test to fail should be so relaxed? It seems to defeat the specificity of the tests.